### PR TITLE
Feat: Meeting materials

### PR DESCRIPTION
**Pull request by kmaid**

### CODE REVIEW

File: apps/storybook/src/AdditionalMaterials.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { AdditionalMaterials } from '@asap-hub/react-components';
+import { number, text } from '@storybook/addon-knobs';
+import { createEventResponse } from '@asap-hub/fixtures';
+
+export default {
+  title: 'Organisms / Additional Materials',
```
Comment (jeysal) [inline apps/storybook/src/AdditionalMaterials.stories.tsx:7]:
```suggestion
  title: 'Organisms / Events / Additional Materials',
```

File: packages/fixtures/src/events.ts [hunk 2]
```diff
@@ -26,9 +27,12 @@ export const createEventResponse = (
   lastModifiedDate: new Date().toISOString(),
   meetingLink: 'https://example.com/meeting',
   notes: 'Meeting notes go here',
-  presentation: 'presentation',
+  presentation: 'Presentation',
   videoRecording: 'Video Recording',
-  meetingMaterials: [],
+  meetingMaterials: Array.from({ length: meetingMaterials }).map((_, i) => ({
+    title: `Material ${i}`,
+    url: '#wrong',
```
Comment (jeysal) [inline packages/fixtures/src/events.ts:34]:
```suggestion
    title: `Material ${i + 1}`,
    url: 'https://example.com/materials/${i}',
```

File: packages/react-components/src/organisms/AdditionalMaterials.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+type AdditionalMaterialsProps = {
+  meetingMaterials: EventResponse['meetingMaterials'];
+};
+const AdditionalMaterials: React.FC<AdditionalMaterialsProps> = ({
+  meetingMaterials,
+}) =>
+  meetingMaterials && meetingMaterials.length ? (
+    <Card>
+      <Headline3 styleAsHeading={4}>Additional meeting materials</Headline3>
```
Comment (jeysal) [inline packages/react-components/src/organisms/AdditionalMaterials.tsx:42]: inconsistent with other parts of the event detail page, which start at heading 2 style as 3 - seems we def need to change the level and you should double check design specs for style

File: packages/react-components/src/organisms/__tests__/AdditionalMaterials.test.tsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  expect(
+    getAllByRole('heading', { level: 4 }).map(({ textContent }) => textContent),
+  ).toEqual(['test', 'test2']);
+});
+
+it('Does not render when meeting materials is empty', () => {
+  const { queryByRole } = render(
+    <AdditionalMaterials {...props} meetingMaterials={[]} />,
+  );
+  expect(queryByRole('heading', { level: 3 })).not.toBeInTheDocument();
```
Comment (jeysal) [inline packages/react-components/src/organisms/__tests__/AdditionalMaterials.test.tsx:41]:
```suggestion
  expect(queryByRole('heading')).not.toBeInTheDocument();
```
harder to cause false pass

File: packages/react-components/src/organisms/__tests__/AdditionalMaterials.test.tsx [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+});
+
+it('Does not render when additional materials is null', () => {
+  const { queryByRole } = render(
+    <AdditionalMaterials {...props} meetingMaterials={null} />,
+  );
+  expect(queryByRole('heading', { level: 3 })).not.toBeInTheDocument();
```
Comment (jeysal) [inline packages/react-components/src/organisms/__tests__/AdditionalMaterials.test.tsx:48]:
```suggestion
  expect(queryByRole('heading')).not.toBeInTheDocument();
```
harder to cause false pass

**Review by jeysal — CHANGES_REQUESTED** [review]
cool mostly lgtm

**Review by jeysal — APPROVED** [review]

**Review by migcarva — APPROVED** [review]

**Pull request merged**